test/e2e/feature/feature.go

```diff
@@ -445,6 +445,10 @@ var (
 	// TODO: document the feature (owning SIG, when to use this feature for a test)
 	StatefulSet = framework.WithFeature(framework.ValidFeatures.Add("StatefulSet"))
 
+	// Owner: sig-node
+	// Denies pod admission if static pods reference other API objects.
+	StaticPodStrictValidation = framework.WithFeature(framework.ValidFeatures.Add("StaticPodStrictValidation"))
```

Comment:
we don't need an e2e feature for this, there's no other configuration associated with it, *if* you write an e2e test, use `framework.WithFeatureGate(features.PreventStaticPodAPIReferences)`, etc